locales/pt_BR.json

```diff
@@ -29,6 +29,14 @@
       "members": {
         "bot": "Bot",
         "user": "Usuário"
+      },
+      "none": "None",
+      "missingAccess": "I am missing access in the channel you linked.",
```

Comment:
Again, why is it in English?

Reply:
It's in English because it's incomplete; translators must do it (or it'll be in English)